content/plugins/diamond.md

```diff
@@ -82,87 +78,15 @@ node_templates:
 # Interfaces
 Two interfaces are involved in setting up a monitoring agent on a machine:
 
-* `cloudify.interfaces.monitoring_agent` - The interface in charge of installing, starting stopping and uninstalling the agent.
-* `cloudify.interfaces.monitoring` - The interface in charge of configuring the monitoring agent.
+* `cloudify.interfaces.monitoring_agent` - The interface that manages installing, starting, stopping, and uninstalling the agent.
+* `cloudify.interfaces.monitoring` - The interface that manages configuring the monitoring agent.
 
 The example above shows how the Diamond plugin maps to these interfaces.
 
-# Global config
```

Comment:
Is this section no longer relevant/incorrect? We seem to be removing a big chunk without putting equivalent info back somewhere else?